config/jail.conf

```diff
@@ -767,3 +767,16 @@ port = 2222
 enabled  = false
 logpath  = /var/lib/portsentry/portsentry.history
 maxretry = 1
+
+[pass2allow-ftp]
+# this pass2allow example allows FTP traffic after successful HTTP authentication
+port         = ftp,ftp-data,ftps,ftps-data
+# knocking URL needs to be customized in apache-pass.local
+filter       = apache-pass
```

Comment:
BTW, what happened with my suggestion (or I email it and it never posted?) to have
```
filter = apache-pass[knocking_url="%(knocking_url)s"]
# TODO:  Define your secret knocking_url in jail.local
knocking_url = 
```
which would use string interpolation to make that jail definition variable available among filter arguments?